Makefile

```diff
@@ -73,23 +73,28 @@ run_ometa_integration_tests:  ## Run Python integration tests
 unit_ingestion:  ## Run Python unit tests
 	coverage run -a --branch -m pytest -c ingestion/setup.cfg --junitxml=ingestion/junit/test-results-unit.xml --ignore=ingestion/tests/unit/source ingestion/tests/unit
 
-.PHONY: coverage
-coverage:  ## Run all Python tests and generate the coverage report
+.PHONY: run_python_tests
+run_python_tests:  ## Run all Python tests with coverage
 	coverage erase
 	$(MAKE) unit_ingestion
 	$(MAKE) run_ometa_integration_tests
+	coverage report || true
+
+.PHONY: coverage
+coverage:  ## Run all Python tests and generate the coverage report
+	$(MAKE) run_python_tests
 	coverage xml -o ingestion/coverage.xml
-	cat ingestion/coverage.xml
+	sed -e 's/$(shell python -c "import site; import os; from pathlib import Path; print(os.path.relpath(site.getsitepackages()[0], str(Path.cwd())).replace('/','\/'))")/src/g' ingestion/coverage.xml >> ingestion/ci-coverage.xml
```

Comment:
I don't like this workaround either. I think here the issue is how we build the `ingestion` project rather than the tool used for processing the coverage reports.

Reply:
in any case when testing this out I also had to update some paths to match the OS in the action agent 🤔 

But yes in any case, we need to cut a bit the noise as we discussed yesterday

Reply:
https://github.com/open-metadata/OpenMetadata/issues/6185

Reply:
I have tested it locally with `make sonar_ingestion` and it works (only for python 3.9)